openedx/features/course_duration_limits/access.py

```diff
@@ -173,17 +174,17 @@ def generate_course_expired_message(user, course):
                 sronly_span_open=HTML('<span class="sr-only">'),
                 span_close=HTML('</span>'),
                 a_close=HTML('</a>'),
-                expiration_date=formatted_expiration_date,
+                expiration_date=HTML(formatted_expiration_date),
```

Comment:
The formatted date should just be plain text, right?

Reply:
To clarify: if it is just supposed to be plain text, we should mark it as such to limit our exposure to malicious translations.

Reply:
Do you mean if this should use Text rather than HTML?  
formatted_expiration_date is a formatted version of the return value of get_date_string which is actually a span, so I think we need the HTML function

Reply:
Ah, ok.